Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -328,6 +424,32 @@ public class SubscribePacket {
                   subscriptionIdentifier: subscriptionIdentifier,
                   userProperties: userProperties)
     }
+
+    typealias RawType = aws_mqtt5_packet_subscribe_view
+    func withCStruct<Result>(_ body: (RawType) -> Result) -> Result {
+        var raw_subscrbe_view = aws_mqtt5_packet_subscribe_view()
+        return self.subscriptions.withCSubscriptions { subscriptionPointer in
+            raw_subscrbe_view.subscriptions =
+                UnsafePointer<aws_mqtt5_subscription_view>(subscriptionPointer)
+            raw_subscrbe_view.subscription_count = self.subscriptions.count
+
+            return withOptionalUserPropertyArray(
+                of: userProperties) { userPropertyPointer in
+
+                    if let userPropertyPointer = userPropertyPointer {
```

Comment:
```suggestion
                    if let userPropertyPointer, userProperties {
```

Reply:
As userProperties is a class member, I dont think it worked, while I tried with following:
```
if let userPropertyPointer,
   let userPropertyCount = self.userProperties?.count {

}
```